modules/administration_manual/pages/configuration/files/encryption/enabling-master-key-encryption.adoc

```diff
@@ -0,0 +1,127 @@
+[[enabling-encryption-from-the-command-line]]
+= Master Key Based Encryption
+Matthew Setter <matthew@matthewsetter.com>
```

Comment:
Why is your contact here, but not above?

Reply:
I likely didn't think to add it, or only started adding it when I came to this file.